Add ISKRA meter images

makuser opened the pull request:
Add images for ISKRA meters

should solve https://github.com/jomjol/AI-on-the-edge-device/issues/689

makuser committed:
Add ISKRA meter images

jomjol commented:
EDIT: found the images and will put them into my next training.
But not all fit to the rule (number central, 20% border)

makuser commented:
> I need the raw data to add it to the training data.

But it is the raw data/picture. I added the raw pictures that I got from the digit log, picked the most representative of them and added them to this merge request. What more would you want to have?

Would you like to add it yourself? Feel free to rewrite the git history if so.

jomjol commented:
Please try the following file: https://github.com/jomjol/AI-on-the-edge-device/raw/rolling/firmware/dig-s0-q-20220224.tflite
